examples/nixnet/CAN_SIGNAL_SINGLE_POINT_OUTPUT.py

```diff
@@ -0,0 +1,87 @@
+
+import grpc
+import nixnet_pb2 as nixnet_types
+import nixnet_pb2_grpc as grpc_nixnet
+import getch
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+
+# Parameters
+INTERFACE = "CAN1"
+DATABASE = "NIXNET_example"
+CLUSTER = "CAN_Cluster"
+SIGNAL_LIST = "CANEventSignal1,CANEventSignal2"
+NUM_SIGNALS = 2
+SUCCESS = 0
+
+# Display Error Function
```

Comment:
Since this a local function, the description of this function should come inside the body. Something like this.

```
def display_error_and_exit(Status, Source)::
"""Display error message and exit."""
... rest of the code
```